scripts/gen.py

```diff
@@ -947,18 +959,27 @@ def generate_registrations(fgens, overrides):
   return code + ';\n}\n', overridden
 
 
+# XLA is only able to override non-compound ops and whitelisted compound ops.
+def is_overrideable(fgen):
+  if not fgen.compound:
+      return True
+  if fgen.mapsig in _FN_WHITELIST or fgen.func in _FN_WHITELIST:
```

Comment:
return ...

Reply:
To double check, is this about the 4 spaces before return? ;)

Reply:
Just return the value of that `if` conditional.